--- fftw-2.1.5/double/configure.in.bero	2013-01-15 21:08:34.610900603 +0100
+++ fftw-2.1.5/double/configure.in	2013-01-15 21:08:55.968643181 +0100
@@ -1,11 +1,11 @@
 dnl Process this file with autoconf to produce a configure script.
-AC_INIT(fftw/planner.c)
-AM_INIT_AUTOMAKE(fftw, 2.1.5)
+AC_INIT(fftw, 2.1.5)
+AM_INIT_AUTOMAKE
 
 dnl This is the version info according to the libtool versioning system.
 dnl It does *not* correspond to the release number.
 SHARED_VERSION_INFO="2:7:0"
-AM_CONFIG_HEADER(fftw/config.h fftw/fftw.h)
+AC_CONFIG_HEADERS([fftw/config.h fftw/fftw.h])
 AM_ENABLE_SHARED(no)
 
 dnl -----------------------------------------------------------------------
--- fftw-2.1.5/single/configure.in.bero	2013-01-15 21:08:59.412602102 +0100
+++ fftw-2.1.5/single/configure.in	2013-01-15 21:09:11.906452530 +0100
@@ -1,11 +1,11 @@
 dnl Process this file with autoconf to produce a configure script.
-AC_INIT(fftw/planner.c)
-AM_INIT_AUTOMAKE(fftw, 2.1.5)
+AC_INIT(fftw, 2.1.5)
+AM_INIT_AUTOMAKE
 
 dnl This is the version info according to the libtool versioning system.
 dnl It does *not* correspond to the release number.
 SHARED_VERSION_INFO="2:7:0"
-AM_CONFIG_HEADER(fftw/config.h fftw/fftw.h)
+AC_CONFIG_HEADERS([fftw/config.h fftw/fftw.h])
 AM_ENABLE_SHARED(no)
 
 dnl -----------------------------------------------------------------------
